src/openrct2/world/Entrance.cpp

```diff
@@ -217,6 +217,23 @@ void fix_park_entrance_locations(void)
         gParkEntrances.end());
 }
 
+void UpdateParkEntranceLocations()
```

Comment:
Rename to `ResetParkEntranceLocations` or something else. Its not really updating them its setting them on first use. Should perhaps be moved to the parkfile as its only used by that.